extensions/ql-vscode/test/vscode-tests/utils/expect-arrays-equal.ts

```diff
@@ -0,0 +1,6 @@
+export function expectArraysEqual<T>(actual: T[], expected: T[]) {
+  // Check that all of the expected values are present
+  expect(actual).toEqual(expect.arrayContaining(expected));
+  // Check that no extra un-expected values are present
+  expect(expected).toEqual(expect.arrayContaining(actual));
+}
```

Comment:
Could we make this a matcher instead located in `test/matchers`? That way, we could call it like `expect(actual).toArrayEquals(expected)`.

Reply:
Hmm, I've opted instead to delete `expectArraysEqual` and use sets instead. The real problem I was trying to solve is to not specify the exact ordering in the tests, because the ordering of the `paths` array shouldn't matter. We can solve that by using `expect(new Set(...)).toEqual(new Set(...))`.

I tried making the `paths` field from `FilePathDiscovery` just be a set but sadly it's just so difficult to work with. A lot of the operations you rely on for arrays just aren't there, like a `has` method that uses a predicate, or `filter`.